crypto/engine/eng_cryptodev.c

```diff
@@ -213,29 +222,31 @@ static struct {
 static struct {
     int id;
     int nid;
-    int keylen;
+    int digestlen;
 } digests[] = {
+#  if 0
+        /* HMAC is not supported */
```

Comment:
yes, but new defines have to be used since as I explained above, CRYPTO_X are enum values.